Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocEnumValue.css

```diff
@@ -89,3 +91,54 @@
     line height remains unchanged. */
   content: "\200b";
 }
+
+.bocEnumValue input[type=radio]
+{
+  z-index: var(--z-index-hidden);
+  position: absolute;
+  opacity: 0;
+}
+
+.bocEnumValue input[type=radio] + label
+{
+  margin-bottom: 0.125em;
+}
+
+.bocEnumValue input[type=radio] + label::before
+{
+  background-image: url('../Image/sprite.svg#RadioButtonFalse');
+  height: var(--default-icon-size);
+  width: var(--default-icon-size);
+  display: inline-block;
+  content: "";
+  margin-bottom: -0.25em;
```

Comment:
are the margins for adjusing the icon due to svg viewbox? we should make them into css variables so this becomes clearer. In particular the .disabled style is a bit lost otherwise.